.github/workflows/docker-ingestion.yml

```diff
@@ -91,3 +92,57 @@ jobs:
           push: ${{ needs.setup.outputs.publish == 'true' }}
           build-args: |
             RELEASE_VERSION=${{ needs.setup.outputs.python_release_version }}
+      - name: Docker meta (slim)
+        id: docker_meta_slim
+        uses: crazy-max/ghaction-docker-meta@v1
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            linkedin/datahub-ingestion-slim
+          # add git short SHA as Docker tag
+          tag-custom: ${{ needs.setup.outputs.tag }}
+          tag-custom-only: true
+      - name: Build and Push image (slim)
+        uses: ./.github/actions/docker-custom-build-and-push
+        with:
+          images: |
+            linkedin/datahub-ingestion-slim
+          tags: ${{ steps.docker_meta_slim.outputs.tags }}
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          publish: ${{ needs.setup.outputs.publish == 'true' }}
+          context: .
+          file: ./docker/datahub-ingestion-slim/Dockerfile
+          platforms: linux/amd64,linux/arm64
+  ingestion-slim_scan:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    name: "[Monitoring] Scan datahub-ingestion-slim images for vulnerabilities"
+    if: ${{ github.ref == 'refs/heads/master' }}
+    runs-on: ubuntu-latest
+    needs: [push_to_registries]
+    steps:
+      - name: Checkout # adding checkout step just to make trivy upload happy
+        uses: actions/checkout@v3
+      - name: Download image
```

Comment:
This required the use of the custom action from the unified workflow. Using this unified workflow custom action in order to hopefully make this work (unlike [this](https://github.com/datahub-project/datahub/pull/7401)).